inference-engine/tools/compile_tool/main.cpp

```diff
@@ -351,7 +351,12 @@ int main(int argc, char *argv[]) {
             outputName = getFileNameFromPath(fileNameNoExt(FLAGS_m)) + ".blob";
         }
         std::ofstream outputFile{outputName};
-        executableNetwork.Export(outputFile);
+        if (!outputFile) {
+            std::cout << "Output file " << outputName << " can't be opened for writing" << std::endl;
+            return EXIT_FAILURE;
+        } else {
+            executableNetwork.Export(outputFile);
```

Comment:
ma be this check should be put into `Export` method itself?

Reply:
In this case it should be implemented in each `Export` function. Also `outputFile` not avaliable there

Reply:
```
    void Export(std::ostream& networkModel) override {
        std::stringstream strm;
        strm.write(exportMagic.data(), exportMagic.size());
        strm << _plugin->GetName() << std::endl;
        ExportImpl(strm);
        networkModel << strm.rdbuf();
    }
```
This place can be changed. All plugins use it.

Reply:
At least 1 plugins override this function. And one more has patch on review to override this function too.

Reply:
ok, let's add this check in both places :)